### ISSUE TITLE
findfeatures - cv::Exception

### ISSUE THREAD

**Issue by lwellerastro** [DOI-USGS/ISIS3#557]
data: /work/users/lweller/Isis3Tests/FindFeatures/cvException

isisversion: isis3.6.0rc and isis3beta2018-09-18 where I initially encountered the problem via a cluster run.  Was hoping this was a node thing....

error:

```
terminate called after throwing an instance of 'cv::Exception'
  what():  /usgs/pkgs/local/v006/src/opencv/opencv-3.1.0/modules/core/src/lapack.cpp:798: error: (-215) type == CV_32F || type == CV_64F in function invert

./run.csh: line 2:  7791 Aborted                 (core dumped) 

```
command:

`findfeatures algorithm=fast/brief maxthreads=7 match=N1500060756_2_callptr.cub fromlist=N1500060756_2_callptr_fromlist.lis fastgeom=true geomtype=camera epitolerance=3.0 ratio=0.65 hmgtolerance=3.0 networkid=N1500060756_2_callptr pointid='N1500060756_2_callptr_?????' onet=N1500060756_2_callptr.net tolist=N1500060756_2_callptr_cubes.lis tonotmatched=N1500060756_2_callptr_notmatched.lis description='Create image-image control network' debug=true debuglog=N1500060756_2_callptr.log`

**Comment by jessemapel** [comment]
This will probably require someone running your data in a debugger to determine where in our code OpenCV is throwing an exception.

At a minimum, we should be trying to catch these and throw a meaningful ISIS exception.

**Comment by lwellerastro** [comment]
Side note for future reference (if this type of error is encountered again): The data that create this problem include images with extremely slim overlaps. While running isisminer to create the input fromlist for containing overlapping images, I had allowed overlap ratio's as small as 0.0055 through.  When I corrected the problem and set the ratio to 0.55 the problem went away.

So the exception is likely being thrown when it encounters overlaps that are so small the detector/extractor algorithm(s) don't have enough information to work with.

**Comment by Kelvinrr** [comment]
Was able to replicate it. The issue seems to be that `findfeatures` passes an empty matrix into `GenericTransform::calculateInverse`. From talking with @lwellerastro she told she got passed this issue a while ago by reprocessing her data. Without a deeper dive with input data she gave `findfeatures`'s state it's impossible to know what exactly caused the problem. I think for now giving a more meaningful error is good enough. 

If this is more than a simple edge case then we should reopen.
